vllm/v1/core/kv_cache_manager.py

```diff
@@ -164,7 +176,9 @@ def allocate_slots(
         self,
         request: Request,
         num_tokens: int,
-        new_computed_blocks: Optional[list[KVCacheBlock]] = None
+        new_computed_blocks: Optional[list[KVCacheBlock]] = None,
+        skip_preallocate: bool = False,
+        skip_inc_ref_count: bool = False,
```

Comment:
If this new arg used to remove a part of this hack, 
https://github.com/vllm-project/vllm/pull/15960/files#r2034074384

Could we add a logic to avoid using a `"temp-req-id-for-connector"` here?

Reply:
yep, working on it